lib/OrganizationsFieldArray/OrganizationCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AppIcon } from '@folio/stripes/core';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty } from 'lodash';
+import {
+  Card,
+  Icon,
+  MultiColumnList,
+  Layout,
+} from '@folio/stripes/components';
+import css from './OrganizationCard.css';
+
+export default class OrganizationCard extends React.Component {
+  static propTypes = {
+    cardStyle: PropTypes.string,
+    headerStart: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    headerEnd: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    id: PropTypes.string,
+    interfaces: PropTypes.object,
+    meta: PropTypes.shape({
```

Comment:
With this and the way you're doing the error handling, you're expressing the dependency that `Card` should a redux form field. This isn't ideal and we don't even always use it like that ourselves, like when we're on the view agreement/license pane.

Reply:
I guess this also loops back to the above comment talking about whether "this component [should] be used when rendering both a selected and unselected organization". 

If it was only used when rendering a _selected_ organization, then you wouldn't need handle displaying errors in this component since the "role" dropdown could always be responsible for it. Then you wouldn't need this dependency on `meta` supplied by redux-form.